web/packages/design/src/DataTable/types.ts

```diff
@@ -43,6 +43,19 @@ export type PaginationConfig = {
   pagerPosition?: 'top' | 'bottom';
 };
 
+// Page keeps track of our current agent list
```

Comment:
I searched the codebase for existing examples and didn't find many. I guess we don't document our typescript code much.

Should we be using something like [tsdoc](https://tsdoc.org) here instead of plain comments?

Reply:
I'm glad you bring this up because our types documentation is quite lacking and a lot of that is due to us using plain comments as opposed to proper syntax that would allow us to document our code better. 

This specific change has been revised, however I think in general we should start documenting our types more and in more useful ways. I filed an issue to track this potential future endeavour: https://github.com/gravitational/teleport/issues/21209